meta/Makefile

```diff
@@ -35,7 +35,6 @@ WARNINGS = \
 	-Werror \
 	-Wextra \
 	-Wextra \
-	-Wfloat-equal \
```

Comment:
don't remove warnings, you are making trouble with that, this warning is for a reason here, and reason is specifically what you are removing it